umap/static/umap/js/modules/help.js

```diff
@@ -190,42 +190,48 @@ export default class Help {
   show(entries) {
     const container = DomUtil.add('div')
     DomUtil.createTitle(container, translate('Help'))
-    // Special dynamic case. Do we still think this dialog is usefull ?
-    if (entries === 'edit') {
+    for (const name of entries) {
       DomUtil.element({
         tagName: 'div',
         className: 'umap-help-entry',
         parent: container,
-      }).appendChild(this._buildEditEntry())
-    } else {
-      for (const name of entries) {
-        DomUtil.element({
-          tagName: 'div',
-          className: 'umap-help-entry',
-          parent: container,
-          innerHTML: ENTRIES[name],
-        })
-      }
+        innerHTML: ENTRIES[name],
+      })
     }
     this.map.dialog.open({ content: container, className: 'dark' })
   }
 
+  // Special dynamic case. Do we still think this dialog is usefull ?
```

Comment:
```suggestion
  // Special dynamic case. Do we still think this dialog is useful?
```

Let's do that in another PR.